package.json

```diff
@@ -31,11 +31,11 @@
   },
   "peerDependencies": {
     "@schedule-x/calendar": "1.50.0",
-    "react": "^16.7.0 || ^17 || ^18",
-    "react-dom": "^16.7.0 || ^17 || ^18"
+    "preact": "^10.22.1"
```

Comment:
I think we can probably pretty safely go with `"preact": "^10"` to allow all minor versions of the current major.